src/mono/wasm/build/WasmApp.targets

```diff
@@ -58,8 +58,10 @@
   -->
 
   <PropertyGroup>
-    <WasmStripAOTAssemblies Condition="'$(AOTMode)' == 'AotInterp'">false</WasmStripAOTAssemblies>
-    <WasmStripAOTAssemblies Condition="'$(WasmStripAOTAssemblies)' == ''">$(RunAOTCompilation)</WasmStripAOTAssemblies>
+    <WasmStripAOTAssemblies>false</WasmStripAOTAssemblies>
+
```

Comment:
Unrelated ?

Reply:
We want to completely disable stripping right now. But yes, can be a separate PR if we want that.

Reply:
Moved it to https://github.com/dotnet/runtime/pull/48500/ .